perk-store-webapps/src/main/webapp/vue-app/components/perk-store/BuyModal.vue

```diff
@@ -92,6 +92,8 @@ export default {
   data() {
     return {
       dialog: false,
+      isSameNetworkVersion: false,
+      polygonURL: 'https://polygon.technology'
```

Comment:
do not hardcode network, this should be retrieved from configuration.
Using the networkID, you can get which network is activated like in https://github.com/meeds-io/wallet/blob/develop/wallet-webapps-common/src/main/webapp/vue-app/js/WalletUtils.js#L299